ygot/diff.go

```diff
@@ -365,6 +365,11 @@ type DiffPathOpt struct {
 	// generated structs, which can result in duplication of list key leaves in
 	// the diff output.
 	MapToSinglePath bool
+	// IgnoreNewPaths specifies that only paths that are set in the original value are
```

Comment:
I think elsewhere we've tried to make there be a single field in the options rather than have multiple -- but I guess `DiffPathOpt` is a fairly generic name. I'd assume that this option should control how paths are interpreted, so we should have a new option that can just be `struct {}` that corresponds to `ModifiedOnly`.

Reply:
Ok, I made it standalone.